tests/testthat/test-mssql.r

```diff
@@ -0,0 +1,46 @@
+context("MSSQL translation")
+
+# Initialize simulated DB connection -------------------------------------
+
+mssql_connection <- dbplyr:::simulate_odbc("Microsoft SQL Server")
```

Comment:
Can you please copy and paste into the individual tests? It's a bit hacky but it makes each test easier to explore interactively in isolation.